### PR TITLE
Add cli spinners

### PR DESCRIPTION
**Pull request by vladimyr**
## Description
Use [`ora`](https://npm.im/ora) powered spinners instead of console logging. :tada:

Check it out in action:

[asciicast](https://asciinema.org/a/l42YYvpas5Jl5IX7MpcsD0PWo)

## Checklist

Please review this checklist before submitting a pull request.

- [x] Code compiles correctly
- [x] Created tests, if possible
- [x] All tests passing (`npm run test:all`)
- [x] Extended the README / documentation, if necessary

### CODE REVIEW

File: lib/tldr.js [hunk 1]
```diff
@@ -140,15 +145,16 @@ function printBestPage(command, options={}) {
         return exit();
       }
       // If not found, try to update
-      console.log('Page not found. Updating cache ..');
+      spinner.start('Page not found, updating cache');
```
Comment (agnivade) [inline lib/tldr.js]: I would prefer to keep 2 sentences. I think grammatically that is more correct. You can remove the ellipses (`...`)

Comment (vladimyr) [reply lib/tldr.js]: `ora` style is without any suffix so I tried to go with that which made me change from period to colon. :grin:
So you ar basicly saying lets use:
`Page not found. Updating cache`?

Comment (agnivade) [reply lib/tldr.js]: Correct.

**Comment by agnivade** [comment]
Thanks !

**Pull request merged**